### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by JONHYOJIN** [ddlBoJack/emotion2vec#72]
Hi, 

Could you share the pretraining code for emotion2vec? (configurations, main training python code, etc.)

**Comment by kokorosawa** [comment]
I think the loss function is also missing
